ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -113,15 +115,24 @@ Quality managed targets are not qualified, but can usually be qualified on reque
      - Full
      - \-
 
+Untested targets
+----------------
 
-Experimental targets
---------------------
-
-Experimental targets cannot be used in safety-critical contexts, and there is
+Untested targets cannot be used in safety-critical contexts, and there is
 no guarantee that the Ferrocene test suite is successfully executed on the
 target. They are provided as a preview, with limited support available. They
 should not be used in production.
 
+For any of the following reasons, the target is not quality managed or qualified:
+
+* The Ferrocene test suite may not successfully execute on the target.
+* Some or all of the Ferrocene test suite does not yet execute on the target.
```

Comment:
Points 1 and 2 sound the same?

Reply:
The intent was that point 1 communicates that the test suite has failing tests for some reason, while point 2 is that some suites cannot be run at all on the target. Let me reword it.